src/pages/using-spark/guides/alt-text.mdx

```diff
@@ -0,0 +1,352 @@
+---
+title: Alternative Text
+---
+
+import { SprkDivider, SprkIcon } from '@sparkdesignsystem/spark-react';
+
+<h1 class="sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure">
+  Alternative Text
+</h1>
+
+UX Writers, designers and developers can use this **Alternative Text Guide** as an instruction to writing style, usage, text for copy, documentation, reference information, and copy for all images inside Rocket Products digital experiences.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvl" />
+
+<h3 class="sprk-b-TypeDisplayThree sprk-u-mbm sprk-u-Measure">Main Takeaways</h3>
+
+- Every image's alternative text should describe the image without altering the meaning of the screen or referencing the word “image.”
+- When writing copy for alternative text, imagine what you would say when reading the screen to someone over the phone without mentioning that there is an image on the screen.
+- Adding alternative text ensures users with cognitive, learning, visual impairments and other disabilities can understand non-text content.
+- Use a null attribute (or empty string) for non-semantic or decorative images, i.e., those that are for ambiance or other non-semantic purposes.
```

Comment:
All of these are too wordy. Let's get to the meaning:
- Adding alternative text ensures user with disabilities can understand non-text content.
- Describe the image without altering the meaning of the screen or referencing the word “image.”
- Consider how you would describe that image to someone without them having to visually see the image.
- Decorative images don't need alternative text.

Does this last point mean you NEED to put `alt=""` even if it doesn't need it?
It could just say 
`Decorative images don't need alternative text`

- Use a null attribute (or empty string) for non-semantic or decorative images, i.e., those that are for ambiance or other non-semantic purposes.